### PR TITLE
Update to deno v1.30.3

### PR DESCRIPTION
**Pull request by GiddyGoatGaming**
Updates deno to 1.30.3

I realized just after I made the pull request to bump (checkout for example) to v3.3.0, it points to the same as if it was v3. So I corrected that.

### CODE REVIEW

**Review by ignoramous — COMMENTED** [review]
Sorry, I missed this commit. Can you pls revive it?

**Comment by GiddyGoatGaming** [comment]
> Sorry, I missed this commit. Can you pls revive it?

Absolutely. I only closed this to change my cloudflare worker repo settings😁

File: src/plugins/cache-util.js [hunk 1]
```diff
@@ -10,8 +10,8 @@ import * as util from "../commons/util.js";
 import * as dnsutil from "../commons/dnsutil.js";
 import * as envutil from "../commons/envutil.js";
 
-const minTtlSec = 30; // 30s
-const maxTtlSec = 180; // 3m
+const minTtlSec = 300; // 30s
+const maxTtlSec = 86400; // 3m
```
Comment (ignoramous) [inline src/plugins/cache-util.js:14]: These are used in DNS packets as-is; which represent these in secs not in millis.

File: src/core/env.js [hunk 2]
```diff
@@ -143,7 +143,7 @@ const defaults = new Map(
     // ttl for dns answers, overrides ttls in dns answers
     CACHE_TTL: {
       type: "number",
-      default: "1800", // 30m
+      default: "86400", // 30m
```
Comment (ignoramous) [inline src/core/env.js:146]: see below (must be in secs not millis)

File: src/core/env.js [hunk 3]
```diff
@@ -121,7 +121,7 @@ const defaults = new Map(
     // secondary doh upstream
     CF_DNS_RESOLVER_URL_2: {
       type: "string",
-      default: "https://dns.google/dns-query",
+      default: "https://dns.nextdns.io/dns-query",
```
Comment (ignoramous) [inline src/core/env.js:124]: perhaps meant for your private repo?

**Review by ignoramous — CHANGES_REQUESTED** [review]
Thanks for reviving the changes.

File: src/core/env.js (hunk 3 above)
Comment (GiddyGoatGaming) [reply src/core/env.js:124]: > perhaps meant for your private repo?

Oh no🤣. I didn’t realize it brought all my commits over here. So sorry about that. Il make a new pull request requesting to update the original stuff I planned to have here when deno 1.31.0 comes out next week. Again, sorry

**Pull request closed without merging**